dragscroll.js

```diff
@@ -90,3 +95,16 @@
     exports.reset = reset;
 }));
 
+function zoomIn(n) {
+    zoom = zoom + n;
+    sheet.innerHTML = "#dragsimg {width:" + zoom + "%;}";
+    document.body.appendChild(sheet);
```

Comment:
I'm not sure if it's a good idea to always re-append the stylesheet. Will it be present multiple times?

(As I wrote, better fetch the elements directly without the stylesheet)